library/src/main/java/com/pokegoapi/api/PokemonGo.java

```diff
@@ -297,7 +304,27 @@ private void initialize() throws RequestFailedException {
 
 		getRequestHandler().sendServerRequests(envelope);
 
-		List<LoginListener> loginListeners = getListeners(LoginListener.class);
+        try {
+            FetchAllNewsMessageOuterClass.FetchAllNewsMessage msg = FetchAllNewsMessageOuterClass.FetchAllNewsMessage.newBuilder().build();
+            ServerRequest request = new ServerRequest(RequestType.FETCH_ALL_NEWS, msg);
+            envelope = ServerRequestEnvelope.create(request);
+            getRequestHandler().sendServerRequests(envelope);
+            FetchAllNewsResponseOuterClass.FetchAllNewsResponse response = FetchAllNewsResponseOuterClass.FetchAllNewsResponse.parseFrom(request.getData());
+            if (response.getResult() == FetchAllNewsResponseOuterClass.FetchAllNewsResponse.Result.SUCCESS) {
+                Log.i(TAG, "FetchAllNewsMessage Success: total News=" + response.getCurrentNews().getNewsArticlesCount());
+                this.news.setCurrentNews(response.getCurrentNews());
+
+                // mark all un-read new to read
+                this.news.markUnreadNews();
+            } else {
+                Log.d(TAG, "FetchAllNewsMessage Failed. Result=" + response.getResult());
+            }
+        } catch (Exception e) {
+            Log.d(TAG, "Exceptions FetchAllNew");
```

Comment:
If this throws an error, it should probably be handled by the caller and not caught like this.